cmd/run.go

```diff
@@ -180,6 +181,22 @@ func runRun(cmd *cobra.Command, newClient ClientFactory) (err error) {
 		return
 	}
 
+	// Check for explicit container=false before applying auto-fix
+	envValue := os.Getenv("FUNC_CONTAINER")
+	containerExplicitlySetToFalse := cmd.Flags().Changed("container") ||
+		(envValue != "" && strings.ToLower(envValue) == "false")
```

Comment:
Theres no need to check the actual ENV value, this is actually handled by the Viper package. You can check this by putting a print statement after the run config is first created.
```go
	cfg = newRunConfig(cmd) // Will add Prompt on upcoming UX refactor
	fmt.Printf("container is %v\n", cfg.Container)
``` 
We already have the "final" container value.